File: PRAKTIKUM 4/PRAK402_2310817220028_SheilaSabina.c
#include <stdio.h>

void main() {
    int angka;

    printf("Input\n");
    scanf("%d", &angka);

    printf("\nOutput\n");
    for (int i = 1; i <= angka; i += 2) {
        printf("%d ", i);
    }

    printf("\n");
    for (int i = angka; i >= 2; i -= 2) {
        printf("%d ", i);
    }

    printf("\n");
}

File: PRAKTIKUM 4/PRAK402_2310817220028_SheilaSabina.py
angka = int(input("Input\n"))

print("\nOutput")

for s in range(1, angka + 1, 2):
    print(f"{s} ", end="")

print()

for s in range(angka, 1, -2):
    print(f"{s} ", end="")

print()